src/main/java/nl/tudelft/dnainator/ui/DNAinator.java

```diff
@@ -24,20 +26,31 @@
 	private static final int MIN_WIDTH = 300;
 	private static final int MIN_HEIGHT = 150;
 
+	private Stage stage;
```

Comment:
I need a reference to set the dimensions in a separate method. Could also just pass the stage, but for #67 it is needed in some other methods as well.